core-bundle/src/Picker/AbstractTablePickerProvider.php

```diff
@@ -136,7 +137,7 @@ public function supportsContext($context): bool
         $this->framework->initialize();
         $this->framework->createInstance(DcaLoader::class, [$table])->load();
 
-        return $this->getDataContainer() === $GLOBALS['TL_DCA'][$table]['config']['dataContainer']
+        return is_a(DataContainer::getDriverForTable($table), $this->getDataContainer(), true)
```

Comment:
Should we allow both versions for better backwards compatibility?
```suggestion
        return ($this->getDataContainer() === $GLOBALS['TL_DCA'][$table]['config']['dataContainer'] || is_a(DataContainer::getDriverForTable($table), $this->getDataContainer(), true))
```
And also update the description of the abstract `getDataContainer` on line 276?

Reply:
I don't get it how this would make a difference 🤔

Reply:
If a custom provider class that extends this abstract provider returns `'Foobar'` in the `getDataContainer()` method and the `$GLOBALS['TL_DCA'][$table]['config']['dataContainer']` is set to `'Foobar'` your version would execute `is_a('DC_Foobar', 'Foobar')` and fail.

Reply:
I see your point now. Thanks!

Reply:
We may add a deprecation for this kind of style?

Reply:
I'll add one.

Reply:
Added in 0963b3dd7.